model/span.go

```diff
@@ -33,9 +33,10 @@ const (
 	// FirehoseFlag is the bit in Flags in order to define a span as a firehose span
 	FirehoseFlag = Flags(8)
 
-	samplerType        = "sampler.type"
-	keySpanKind        = "span.kind"
-	samplerTypeUnknown = "unknown"
+	samplerType              = "sampler.type"
+	keySpanKind              = "span.kind"
+	samplerTypeUnknown       = "unknown"
+	samplerTypeProbabilistic = "probabilistic"
```

Comment:
These could be made public, since they are returned from the public method GetSamplerType, and you need at least one of them in the aggregator code.

An even better solution is to make this strongly typed:
```
type SamplerType int

const (
    SamplerTypeUnknown SamplerType = iota
    SamplerTypeProbabilistic
}

func (s *Span) GetSamplerType() SamplerType {}
```